src/test/java/org/jenkinsci/plugins/workflow/SerializationTest.java

```diff
@@ -233,6 +233,34 @@ public void stop(Throwable cause) throws Exception {
         });
     }
 
+    @Issue("JENKINS-26481")
+    @Test public void eachClosureNonCps() {
```

Comment:
Verifies that we are not throwing the `UnsupportedOperationException` too aggressively:

* on non-CPS-transformed `Closure`s
* on closures passed to methods defined in Pipeline script
* on closures passed to methods which did not declare `Closure` as a parameter type and so presumably are not going to try to call them